src/disco/fakeData.js

```diff
@@ -7,19 +12,21 @@ export default {
     slug: 'japanese-tattoo',
     sub_heading: null,
     type: 'Theme',
-    url: 'https://addons-dev.allizom.org/en-US/firefox/addon/japanese-tattoo/',
-    headerURL: 'https://addons-dev-cdn.allizom.org/user-media/addons/18781/personare.jpg?1239806327',
-    footerURL: 'https://addons-dev-cdn.allizom.org/user-media/addons/18781/persona2re.jpg?1239806327',
+    url: `${amoHost}/en-US/firefox/addon/japanese-tattoo/`,
+    themeURL: `${amoCDN}/user-media/addons/18781/preview_large.jpg?1239806327`,
```

Comment:
I made it consistent with the other keys. We may as well adapt it to the data all in one go.